playbooks/robusta_playbooks/babysitter.py

```diff
@@ -79,3 +82,27 @@ def resource_babysitter(event: KubernetesAnyChangeEvent, config: BabysitterConfi
         title="Kubernetes Manifest Change",
     )
     event.add_finding(finding)
+
+
+class UrlParam(ActionParams):
+    """
+    :var url: url that should be used in the action
+    """
+
+    url: str
+    headers: Optional[Dict[str, str]] = None
+
+
+@action
+def json_change_tracker(event: KubernetesAnyChangeEvent, params: UrlParam):
+    """
+    post change json to the specified url
+    this action doesn't create a finding
+    """
+    try:
+        event_dict = event.obj.to_dict()
```

Comment:
is this enough?
do we want event.obj and event.old_obj to see the changes? or only the new one?
event.filtered_diffs, event.obj_filtered, event.old_obj_filtered?

If they only want the json of the new object and not the changes wouldn't it make more sense to use the webhook sink and then make a get_resource_json action like what we have for get_resource_yaml?

Reply:
yes, in this case they just want to resource json, without the finding details
so this simple solution works best here